sources/Launcher/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@
 
 set(DYN_ALGO_LAUNCHER_SOURCES
   DYNComputeLoadVariationLauncher.cpp
+  DYNComputeSimulationLauncher.cpp
```

Comment:
.h is not necessary to install ? in DYN_ALGO_LAUNCHER_HEADERS below

Reply:
solved